build.sbt

```diff
@@ -9,10 +9,13 @@ lazy val rootProject = (project in file(".")).settings(
     libraryDependencies ++= Seq(
       "com.softwaremill.sttp.tapir" %% "tapir-zio-http-server" % tapirVersion,
       "com.softwaremill.sttp.tapir" %% "tapir-json-zio" % tapirVersion,
-      "ch.qos.logback" % "logback-classic" % "1.4.5",
+      "com.softwaremill.sttp.tapir" %% "tapir-swagger-ui-bundle" % tapirVersion,
       "com.softwaremill.sttp.tapir" %% "tapir-sttp-stub-server" % tapirVersion % Test,
-      "dev.zio" %% "zio-test" % "2.0.5" % Test,
-      "dev.zio" %% "zio-test-sbt" % "2.0.5" % Test,
+      "dev.zio" %% "zio-logging" % "2.1.8",
+      "dev.zio" %% "zio-logging-slf4j" % "2.1.8",
```

Comment:
[minor] probably a good idea to extract versions numbers of artifacts that are deployed together to a constant